signingscript/docker.d/passwords.yml

```diff
@@ -144,6 +144,12 @@ in:
              {"$eval": "AUTOGRAPH_AUTHENTICODE_PASSWORD"},
              ["autograph_authenticode", "autograph_authenticode_stub"]
             ]
+          - ["https://autograph-external.stage.autograph.services.mozaws.net",
+             {"$eval": "AUTOGRAPH_MOZILLAVPN_DEBSIGN_USERNAME"},
+             {"$eval": "AUTOGRAPH_MOZILLAVPN_DEBSIGN_PASSWORD"},
+             ["autograph_debsign"],
+             "dummy_gpg2_pgpsubkey_debsign"
```

Comment:
I believe the keyid is default, so we don't need to specify, but we can. If we do specify, this would be `release_at_mozilla_debsign_dep`.